packages/effective-module-tree/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## 0.1.0 - 2020-04-14
```

Comment:
Should we also update the version in our package.json files?

Reply:
As soon as I get this merged, I'll do a release that will upload the version in `package.json`.

Reply:
Actually you were right.
According to `docs/monorepo.md` I should have bumped package.json and merge it to master before trying to release it.